theano/gpuarray/tests/test_subtensor.py

```diff
@@ -85,3 +86,42 @@ def test_incsub_f16():
     rep = xval.copy()
     rep[1:] += yval
     assert numpy.allclose(rval, rep)
+
+
+class G_advancedsubtensor(test_subtensor.TestAdvancedSubtensor):
+    def shortDescription(self):
+        return None
+
+    def __init__(self, name):
+        test_subtensor.TestAdvancedSubtensor.__init__(
+            self, name,
+            shared=gpuarray_shared_constructor,
+            sub=GpuAdvancedSubtensor,
+            inc_sub=GpuIncSubtensor,
```

Comment:
The superclass doesn't have a dimshuffle parameter.  I also noticed that inc_sub should be AdvancedIncSubtensor so I removed it.